client/src/templates/Challenges/components/scene/scene.tsx

```diff
@@ -21,11 +21,32 @@ export function Scene({ scene }: { scene: FullScene }): JSX.Element {
     new Audio(`${sounds}/${audio.filename}${audioTimestamp}`)
   );
 
+  const loadImage = (src: string) => {
+    const img = new Image();
+    img.src = src;
```

Comment:
```suggestion
  const loadImage = (src: string | null) => {
    if (src) new Image().src;
```
It's a little cleaner and lets us tidy up below.